File: vbnet/App_Code/DefaultController.vb

Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports WebEx.Core

''' <summary>
''' Summary description for WebExController
''' </summary>
Public Class DefaultController
    Inherits Controller
    '
    ' TODO: Add constructor logic here
    '
    Public Sub New()
    End Sub

    Public Function index(Optional page As String = "", Optional id As String = "") As ActionResult
        LoadModules(page, id)
        Return View()
    End Function
End Class

File: csharp-del/App_Code/DefaultController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

/// <summary>
/// Summary description for WebExController
/// </summary>
public class DefaultController : Controller
{
    public DefaultController()
	{
		//
		// TODO: Add constructor logic here
		//
	}

    public ActionResult index(string page = "", string id = "")
    {
        WebExModel model = new WebExModel();
        model.Load(this, page, id);
        return View(model);
    }
}